.idea/usage.statistics.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
@mecha, could you explain what this file is, and why it is good for it to be a part of the project?

Reply:
Mistake. PHPS is adding them out of the blue for me. Will remove.